external_parser/parse_example_external.cc

```diff
@@ -38,6 +39,9 @@ parser::get_external_parser(vw *all, const input_options &parsed_options) {
     } else {
```

Comment:
what about:

```
if (binary_to_json)
{
      joiner = ...
} else if (multistep)
{
   joiner = VW::make_unique<multistep_example_joiner>(all);
}
else
{
    joiner = VW::make_unique<example_joiner>(all);
}
```

from my understanding to enable this we need to pass both `--binary_parser` and `--multistep` right?

Reply:
yes, that is true